src/Lib/Assert/Text.php

```diff
@@ -37,4 +37,23 @@ public static function hasMin(int $lim, string $str): bool
     {
         return strlen($str) >= $lim;
     }
+
+
+    /**
+     * Test if string is date
+     *
+     * @return bool
+     */
+    public static function isTimestamp(string $str): bool
+    {
+        $ts = \App\Lib\Util\Timestamp::timestampToArray($str);
+        if (checkdate($ts['m'], $ts['d'], $ts['y']) &&
```

Comment:
I think you can validate timestamp format by https://www.php.net/manual/en/function.strtotime.php it returns false if format is not valid.

Reply:
fyes can be, but i dont think i could check for yyyy-mm-ddThh:ii and pass it accordingly, also we'd loose the ability of easy checks for separate values, like:
```
$ts = \App\Lib\Util\Timestamp::timestampToArray($str);
$hour = $ts['i'];
```

Reply:
It is true, but that is meant that 2021-01-01 will not be treted as valid timestamp despite the fact it can be converted into unix timestamp. Same with passing only time.